templates/base.html.twig

```diff
@@ -17,6 +17,11 @@
 		{% endblock %}
 	</head>
 	<body>
-		{% block body %}{% endblock %}
+		<main>
+			<div class="container">
+				{% block body %}{% endblock %}
+			</div>
+			{% include 'components/_footer.html.twig' %}
+		</main>
```

Comment:
Isn't it strange to have the footer in the main ?
You could have a main (and skip the div) and a footer after it